system-int-test/test/cmr/system_int_test/ingest/provider_ingest_test.clj

```diff
@@ -12,10 +12,12 @@
     [cmr.system-int-test.data2.granule :as dg]
     [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
     [cmr.system-int-test.system :as s]
+    [cmr.system-int-test.utils.association-util :as au]
```

Comment:
`au` -> `association-util`

Reply:
The namespace is not relevant enough to be spelled out. We have been using `au` in other tests as well to make the code more readable.